commec/config/result.py

```diff
@@ -260,28 +261,52 @@ def update_query_flag(self):
             self.benign_status
         }:
             self.screen_status = ScreenStatus.ERROR
+            self.rationale = "There was an error during screening"
+            return
+                
+        if self.benign_status == ScreenStatus.CLEARED_FLAG:
+            self.screen_status = self.benign_status
+            self.rationale = "A sequence of concern was cleared as benign"
             return
 
-        if self.benign_status not in {
-            ScreenStatus.NULL, 
-            ScreenStatus.SKIP
-        }:
+        if self.benign_status == ScreenStatus.CLEARED_WARN:
+            self.screen_status = self.benign_status
+            self.rationale = "A mixed sequence of concern was cleared as benign"
+            return
+
+        if self.benign_status == ScreenStatus.PASS:
             self.screen_status = self.benign_status
+            self.rationale = "No regulated regions of concern"
             return
 
         if self.biorisk_status == ScreenStatus.FLAG:
             self.screen_status = ScreenStatus.FLAG
+            self.rationale = "A known biorisk was detected"
```

Comment:
I know we eventually want to propagate up more detail (toxin? viral virulence factor?) but for now maybe
```suggestion
            self.rationale = "Matched a sequence with pathogenic or toxic function "
```